reference/replica-operations.md

```diff
@@ -2,27 +2,26 @@
+The control plane will first 'retire' the old, faulted one which will then no longer be associated to the volume. Once retired, a replica will become available for garbage collection (deletion from the Mayastor Pool containing it), assuming that the nature of the failure was such that the pool itself is still viable (i.e. the underlying disk device is still accessible).  N.B.: For a replica to be retired from a volume, it must have been possible to create a suitable replacement first.  In the absence of a replacement, the replica remains a member of the nexus, albeit faulted and thus unusable.
+ Then it will attempt to restore the desired state \(replica count\) by creating a new replica, following its replica placement rules.  If it succeeds, the nexus will "rebuild" that new replica - performing a full copy of all data from a healthy replica `Online`, i.e. the source.  This process can proceed whilst the volume continues to process application I/Os although it will contend for disk throughput at both the source and destination disks.     
 
-If a nexus is restarted, i.e. the Mayastor pod hosting it restarts, with the assistance of the control plane it will 'recompos' itself; all of the previous member replicas will be re-attached to it, if possible, including any that were in the state `CHILD_FAULTED`.  If faulted replicas can be re-connected successfully, then the control plane will attempt to rebuild them directly, rather than seek replacements for them first.  This edge-case therefore does not result in the retirement of the affected replicas; they are simply reused.  If they are successfully re-attached but then continue to encounter I/O failures the rebuild will fail and will not be attempted again.
+If a nexus is restarted, i.e. the Mayastor pod hosting it restarts, with the assistance of the control plane it will 'recompose' itself; all of the previous healthy member replicas will be re-attached to it.  If previously faulted replicas are available to be re-connected (`Online`), then the control plane will attempt to reuse and rebuild them directly, rather than seek replacements for them first. This edge-case therefore does not result in the retirement of the affected replicas; they are simply reused.  If the rebuild fails then we follow the above process of removing a `Faulted` replica and adding a new one.
```

Comment:
It's not necessarily true that all previous healthy members will be re-attached. If the `clean shutdown` flag is not set (in the case of an unexpected pod restart) only a single healthy replica will be selected when recreating the nexus.